status-im/status-react
# [15660] Show who sent message reaction

fixes #15660

### Summary
This PR adds the ability to see who reacted to message by long-pressing on a reaction.

### Testing notes
Kindly test this in communities and different types of chats.

#### Platforms

- Android
- iOS


#### Areas that maybe impacted


##### Functional

- 1-1 chats
- public chats
- group chats

### Steps to test

- Open Status
- Send a message
- React to it
- Long press on a reaction to see who reacted to it

https://user-images.githubusercontent.com/33176106/233290710-15a072e3-af6b-424f-bb79-f02ec71864fe.MOV

status: ready
